app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -269,8 +269,10 @@ private void updateCallToAction() {
     }
 
     private void initArticleHeaderView() {
-        pageHeaderView.addOnLayoutChangeListener((View v, int left, int top, int right, int bottom,
-                                                  int oldLeft, int oldTop, int oldRight, int oldBottom) -> setWebViewPaddingTop());
+        pageHeaderView.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
+            LeadImagesHandler.this.setWebViewPaddingTop();
+            parentFragment.getWebView().setVisibility(VISIBLE);
```

Comment:
Would it be more logical to put this in `onPageFinished()` in PageFragment?

Reply:
Changes done @dbrant